frontend-react/src/components/header/AdminDropdownNav.tsx

```diff
@@ -34,7 +35,7 @@ export const AdminDropdownNav = () => {
                 isCurrent={isOpen}
             />
             <Menu
-                items={testMenuItems}
+                items={adminMenuItems}
```

Comment:
just a question...
this is supposed to show for users are not admins, but it is put in the admin section of the menu, right?
does this option show up for non-admins?

Reply:
These menu items should NOT show up for regular users.

The page can still be reached by regular users who are logged in by going directly to the /features page. (This is intended to test features with specific end users, but not have it be generally discoverable.) 

But finding the page is still not enough... the user must know what the flag string needs to be added.